### DISCUSSION TITLE
v4.0.26，登录跑左边了

### DISCUSSION THREAD

**Discussion by kaoding** [umijs/umi#9577] [General]
![image](https://user-images.githubusercontent.com/18542211/196891743-55afb070-453a-4db0-9054-e925570872f2.png)

**Comment by kaoding** [comment]
v4.0.25刚改好一个bug，v4.0.26又出新bug，认真测试一下。

**Comment by xiaohuoni** [comment]
啥左边啊？没看懂

**Comment by kaoding** [comment]
antd pro 登录页面跑左边了（本来是中间的）。

**Comment by xiaohuoni** [comment]
可是你截的图不是 移动端的嘛？pro 又没有做移动端适配！

**Reply by kaoding** [reply]
不是移动端的，是PC端的。antd pro代码登录界面在4.0.26就在左边了，改为4.0.24就是在中间（正常）

**Comment by fz6m** [comment]
解决方案：

1. 删除 `src/e2e` 目录。

2. 或者添加如下配置：

    ```ts
    // .umirc.ts
    
    export default {
      mfsu: {
        exclude: ['@playwright/test']
      }
    }
    ```

FYI : 

 - https://github.com/umijs/umi/issues/9568
